MAUI/Maps/data-labels.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 Data labels provides identification for the shapes by displaying their names. You can trim or hide the labels if they exceed the shape bounds.
 
+To learn more about our .NET MAUI Maps data labels, you can check the below video.
```

Comment:
To learn more about the .NET MAUI Maps data labels, you can check the following video.